### PR TITLE
[oracle_weblogic] Add Integration Package with Deployed Application Data Stream

### PR DESCRIPTION
**Pull request by kush-elastic**
- Enhancement

## What does this PR do?

- Generated the skeleton of Oracle WebLogic integration package.
- Added 1 data stream (Deployed Application).
- Added data collection logic for the data streams.
- Added the ingest pipeline for the data streams.
- Mapped fields according to the ECS schema and added Fields metadata in the appropriate yaml files.
- Added dashboards and visualizations.
- Added system test cases for the data stream.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/oracle_weblogic directory.
- Run the following command to run tests.
`elastic-package test`

## Related issues

- Relates #2823
- Relates #2910
- Relates #3066
- Relates #3067
- Relates #3068
- Relates #3069
- Relates #3189

## Screenshots
image

### CODE REVIEW

File: packages/oracle_weblogic/data_stream/deployed_application/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - rename:
+      field: oracle_weblogic.deployed_application.deployment.state
+      target_field: oracle_weblogic.deployed_application.deployment.state_value
+      ignore_missing: true
+      ignore_failure: true
+  - set:
+      field: oracle_weblogic.deployed_application.deployment.state
+      value: Shutdown
+      ignore_empty_value: true
+      if: ctx.oracle_weblogic.deployed_application.deployment.state_value == 0
```
Comment (yug-elastic) [inline packages/oracle_weblogic/data_stream/deployed_application/elasticsearch/ingest_pipeline/default.yml:26]: Using a painless script with a `HashMap` seems to be a better option here.

File: packages/oracle_weblogic/data_stream/deployed_application/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      ignore_empty_value: true
+      if: ctx.oracle_weblogic.deployed_application.deployment.state_value == 1
+      ignore_failure: true
+  - set:
+      field: oracle_weblogic.deployed_application.deployment.state
+      value: Running
+      ignore_empty_value: true
+      if: ctx.oracle_weblogic.deployed_application.deployment.state_value == 2
+      ignore_failure: true
+  - remove:
```
Comment (yug-elastic) [inline packages/oracle_weblogic/data_stream/deployed_application/elasticsearch/ingest_pipeline/default.yml:40]: Please move `remove` to the end

**Review by yug-elastic — CHANGES_REQUESTED** [review]
Requested some changes

**Review by yug-elastic — APPROVED** [review]
LGTM!

**Review by agithomas — APPROVED** [review]
Looks good to me

**Pull request merged**